htdocs/multicurrency/multicurrency_rate.php

```diff
@@ -111,46 +117,46 @@
 		$currencyRate_static->rate = $rateinput;
 
 		$result = $currencyRate_static->create(intval($fk_currency));
-		if ($result > 0) {
-			setEventMessages($langs->trans('successRateCreate', $multicurrency_code), null);
+		if ($result) {
+			setEventMessage($langs->trans('successRateCreate', $multicurrency_code));
 		} else {
 			dol_syslog("currencyRate:createRate", LOG_WARNING);
-			setEventMessages($currencyRate_static->error, $currencyRate_static->errors, 'errors');
+			setEventMessage($langs->trans('successRateCreate'));
```

Comment:
Why replacing the setEventMessages that propagate the ->error and ->errors into setEventMessage with a success message here ?

Reply:
missed this one , thanks.